bin/idl.js

```diff
@@ -762,6 +762,9 @@ function update(cb) {
 
     function onMeta(err, meta) {
         if (err) {
+            if (err.constructor.name === 'SyntaxError') {
```

Comment:
can we do this the opposite way -- instead of blacklisting SyntaxError, if we really care about "file not found" being OK, then we should whitelist just that single error.

Otherwise, if there's some other error with meta.json (e.g. permissions), then it'll fail silently again.